Dockerfile

```diff
@@ -55,7 +50,6 @@ ONBUILD COPY . $APP_SOURCE_DIR
 # install all dependencies, build app, clean up
 ONBUILD RUN cd $APP_SOURCE_DIR && \
   $BUILD_SCRIPTS_DIR/install-deps.sh && \
-  $BUILD_SCRIPTS_DIR/install-node.sh && \
```

Comment:
If the file `scripts/install-node.sh` is no longer necessary, you could delete it.

Reply:
And you may want to remove the references to custom Node version installations in the `README.md` file.

Reply:
Done

Reply:
Done